code/modules/mob/abstract/new_player/new_player.dm

```diff
@@ -66,9 +56,19 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 			else
 				output += "<a href='byond://?src=\ref[src];showpoll=1'>Show Player Polls</A>"
 
+	if(character_name)
+		output += "<hr>You will board as <b>[character_name]</b><br>"
+	if(ROUND_IS_STARTED)
+		output += "<a href='byond://?src=\ref[src];late_join=1'>Join the Game!</A> "
+	else
+		if(ready)
+			output += "<a href='byond://?src=\ref[src];ready=0'>Un-Ready</a> "
+		else
+			output += "<a href='byond://?src=\ref[src];ready=1'>Ready Up</a> "
+
```

Comment:
No, this looks terrible and unaligned with the rest if you put it on a separate line

Reply:
I think its better because it makes it more obvious since it's more important

Reply:
Also the player polls thing would probably screw with it if they were all there at the same time

![JlWlWHw7MJ](https://user-images.githubusercontent.com/42310821/83046358-7bfebb00-a079-11ea-90f2-34caa6056f51.png)

i was trying to avoid a situation like this with our current one

Reply:
I have no idea what you're trying to say I'm not gonna lie

Reply:
![pWy46ICeKR](https://user-images.githubusercontent.com/42310821/83594868-eee4c600-a592-11ea-9d81-d7af57abb845.png)

I just liked how the ready up thing was in its on level